pkg/providers/aws/provider_redis.go

```diff
@@ -130,7 +129,8 @@ func createRedisCluster(cacheSvc elasticacheiface.ElastiCacheAPI, redisConfig *e
 			logrus.Info("found existing redis cluster")
 			primaryEndpoint := foundCache.NodeGroups[0].PrimaryEndpoint
 			connData := map[string][]byte{
-				"connection": []byte(fmt.Sprintf("%s:%s", *primaryEndpoint.Address, strconv.FormatInt(*primaryEndpoint.Port, 10))),
+				"uri":  []byte(*primaryEndpoint.Address),
```

Comment:
To standardise this format we could move this out to a struct, similar to what was done for the SMTP credentials. wdyt?

https://github.com/integr8ly/cloud-resource-operator/blob/52eb01c7bd6f3b8618027054e033c57c755da4cb/pkg/providers/types.go#L29

https://github.com/integr8ly/cloud-resource-operator/blob/master/pkg/providers/aws/provider_smtpcredentialset.go#L36-L52

Reply:
Oh, I'll update the postgres PR to follow that pattern as well

Reply:
Yeah sounds good :+1: